src/Service/NotificationService.php

```diff
@@ -94,35 +84,76 @@ public function __construct(
      */
     public function notificationHandler(array $data, array $configuration): array
     {
-        if ($data['method'] !== 'POST') {
-            return $data;
-        }
+        $this->logger->debug('NotificationService -> notificationHandler()');
+
+        // Check if we have a method and is POST or GET.
+        if (isset($data['method']) === false || in_array($data['method'],  ['POST', 'GET']) === false) {
```

Comment:
Why allow GET here?
If we want to see all 'notifications' created returning $data should be enough I think 🤔

Reply:
Sort of agree with Wilco, but I can unsderstand that there are some notification systems that, somehow, manage to fire GET requests with the data in the query or something like that...